modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapStyles.vue

```diff
@@ -57,26 +58,80 @@ export default {
       }
     }
   },
+  data() {
+    return {
+      multipleSelected: false
+    };
+  },
   computed: {
+    nodeOptions() {
+      return [ {
+        label: this.$t('apostrophe:richTextNodeMultipleStyles'),
+        attr: this.multipleSelected ? '' : 'hidden'
+      },
+      ...this.options.nodes ];
+    },
     active() {
-      for (let i = 0; (i < this.options.nodes.length); i++) {
-        const style = this.options.nodes[i];
-        if (this.editor.isActive(style.type, (style.options || {}))) {
-          return i;
-        } else if (this.editor.state.selection.$head.parent.type.name === 'defaultNode' && style.def) {
-          // Look deeper to see if custom defaultNode is active
-          return i;
+      const { selection } = this.editor.state;
+      const content = selection.content();
+      let activeEls = [];
+      const nodes = this.options.nodes.map(n => {
+        return {
+          type: n.type,
+          class: n.options.class || null,
+          level: n.options.level || null
+        };
+      });
+
+      if (content?.content?.content.length) {
```

Comment:
If you're sure `content.content` is enough to imply `content.content.content` will always be there. (Not that I know what any of this is)